autoscale-core/src/main/java/com/hpe/caf/autoscale/core/AutoscaleScheduler.java

```diff
@@ -25,18 +28,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
Can we leave these as they were

Reply:
Oh, sorry, I missed this, email arrived much later for some reason.  It will have to be a separate commit now, still worth it?